.env

```diff
@@ -1,2 +1,3 @@
 PORT=3000
 CHOKIDAR_USEPOLLING=true
+REACT_APP_GITHUB_TOKEN=37fe0fb6ffd68943576cdb020d456db2ebb2176e
```

Comment:
No need to panic! I've removed the oauth token no longer valid.

Also, why isn't `.env` in .gitignore -_-

Reply:
well, this repo wasn't made using create-react-app it's missing a lot of things